Update to License, Notice and Readme files

TheJuanAndOnly99 opened the pull request:
~~- JP Morgan Chase should be on the LICENSE and NOTICE files~~
- FINOS "active" and OpenSSF "passing" badges
- Contributing + License sections added to readme

texodus commented:
@TheJuanAndOnly99 Thanks for the PR!

You can't reassign the copyright without the consent of the current copyright holders, e.g. the `regular-table` contributors.  Why do we want to reassign the copyright?

texodus commented on README.md:
```diff
@@ -556,3 +558,27 @@ yarn start
+To submit a contribution:
+1. Fork it (<https://github.com/finos/regular-table/fork>)
+2. Create your feature branch (`git checkout -b feature/fooBar`)
+3. Read our [contribution guidelines](./CONTRIBUTING.md) and [Community Code of Conduct](https://www.finos.org/code-of-conduct)
```
Dead link

timkpaine replied:
wffm

<img width="1079" alt="Screenshot 2023-01-01 at 3 28 52 PM" src="https://user-images.githubusercontent.com/3105306/210183788-2c77390b-e693-48c8-901d-7d9181505d8b.png">

timkpaine commented on README.md:
```diff
@@ -556,3 +558,27 @@ yarn start
 ## Stats
 ![npm bundle size](https://img.shields.io/bundlephobia/minzip/regular-table) 
 -->
+
```
This is already in the [CONTRIBUTING.md](https://github.com/finos/regular-table/blob/master/CONTRIBUTING.md) which has been the convention for programming project for decades.

TheJuanAndOnly99 replied:
removed

texodus reviewed (changes requested):
Can't reassign copyright - need the consent of the original authors (which I grant but there are over a dozen others).

I really think `README.md` stickers are tacky looking but whatever is required I'll add - but the contributing guidelines need some cleaning (and IMO are pretty superfluous ...).

EDIT nvm I'm wrong, this is not a dead link; but still, Github literally inlines these same instructions next to the "fork" button.

timkpaine commented on README.md:
```diff
@@ -556,3 +558,27 @@ yarn start
+
+*Need an ICLA? Unsure if you are covered under an existing CCLA? Email [help@finos.org](mailto:help@finos.org)*
+
+## License
```
This provides no additional useful information on the README while occupying additional screen space. 

Github already puts that on the screen:
<img width="372" alt="Screenshot 2023-01-01 at 3 16 19 PM" src="https://user-images.githubusercontent.com/3105306/210183531-22934a70-3e13-49e5-8c77-6cad1b615e6d.png">

and more information can be gleaned from the LICENSE file which has been the convention for...you guessed it, decades.

timkpaine commented:
unrelated but this looks bad, "openssf best practices: passing" is information sparse given the pixel footprint it takes up. I don't think this badge is necessary and I think it's distracting. 

<img width="640" alt="Screenshot 2023-01-01 at 3 14 26 PM" src="https://user-images.githubusercontent.com/3105306/210183486-4074fc25-6d33-4a0a-b6d7-1ffad13abf13.png">

TheJuanAndOnly99 commented:
Hi @timkpaine @texodus ! Thanks for your feedback I think you raise great points. I have:
- Updated the PR to revert the copyright to how it was. 
- Removed the contributing section from the readme and updated the license section (What do you think of having an AUTHORS file like in [jupyter_templates](https://github.com/finos/jupyterlab_templates/blob/main/AUTHORS)?)
- Removed the OpenSSF badge. Though we would like it to be mentioned somewhere in the readme that this project achieves the OpenSFF passing badge with a link to the project page on the openSFF best practices website https://bestpractices.coreinfrastructure.org/en/projects/6771 (I'll leave it up to you on where we can put this).

Thank you and please let me know what you think!

timkpaine commented:
> * Removed the contributing section from the readme and updated the license section (What do you think of having an AUTHORS file like in [jupyter_templates](https://github.com/finos/jupyterlab_templates/blob/main/AUTHORS)?)

@maoo and i like the `AUTHORS` file approach, im sure @texodus won't mind.

> Removed the OpenSSF badge. Though we would like it to be mentioned somewhere in the readme that this project achieves the OpenSFF passing badge with a link to the project page on the openSFF best practices website https://bestpractices.coreinfrastructure.org/en/projects/6771 (I'll leave it up to you on where we can put this).

maybe in the [development](https://github.com/finos/regular-table#development) block is ok

TheJuanAndOnly99 commented:
> @maoo and i like the `AUTHORS` file approach, im sure @texodus won't mind. 

I've added a AUTHORS file. Feel free to send me list of authors or modify as needed.

> maybe in the [development](https://github.com/finos/regular-table#development) block is ok

Added a small section in the development block

TheJuanAndOnly99 commented:
Hi @texodus @timkpaine thank you for your collaboration. This is good to go on our end, feel free to merge when ready.

timkpaine commented on README.md:
```diff
@@ -3,6 +3,7 @@
 </p>
 
 <p align="center">
+<a href="https://community.finos.org/docs/governance/software-projects/stages/active/"><img alt="OpenSFF badge" src="https://cdn.jsdelivr.net/gh/finos/contrib-toolbox@master/images/badge-active.svg"></a>
```
alt text is wrong

TheJuanAndOnly99 replied:
Good catch, fixed.
